### PR TITLE
q41

### PR DESCRIPTION
**Pull request by Navleenkaursandhu**
q41:
image

### CODE REVIEW

File: Javascript/q41.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q41</title>
+
+<script>
```
Comment (Mantaseus) [inline Javascript/q41.html:9]: Fix indentation on this block. Please

File: Javascript/q41.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  function myFunction() {
```
Comment (Mantaseus) [inline Javascript/q41.html:10]: What is `myFunction`? It doesn't tell me anything useful about what this function does.

File: Javascript/q41.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    document.getElementById("element1").innerHTML = Math.floor(Math.random() * 100);
+  }
+</script>
+</head>
+
+<body>
+  <button onclick="myFunction()">Generate random number between 0 and 100 </button>
+  <p id="element1"></p>
```
Comment (Mantaseus) [inline Javascript/q41.html:18]: `element1` is not a descriptive ID

Comment (Navleenkaursandhu) [reply Javascript/q41.html:18]: done the changes

File: Javascript/q41.html (hunk 2 above)
Comment (Navleenkaursandhu) [reply Javascript/q41.html:10]: done the changes

**Review by Mantaseus — APPROVED** [review]

**Pull request merged**